### ISSUE TITLE
WCAG 1.3.1: <ul> and <ol> must only directly contain <li>, <script> or <template> elements (.card-list)

### ISSUE THREAD

**Issue by zcjess** [zcjess/Problem-Roulette#8]
#### Issue
`<ul> and <ol> must only directly contain <li>, <script> or <template> elements` ([list](https://accessibilityinsights.io/info-examples/web/list))
#### Target application
[Problem Roulette](https://staging.problemroulette.ai.umich.edu/AES220/SessionHistory/156388/)
#### Element path
.card-list
#### Snippet
`<ul class="card-list questions">`

Fix the following:List element has direct children that are not allowed: div
--

![image](https://github.com/zcjess/Problem-Roulette/assets/90011962/566e8673-079d-4972-8251-a48942dfd5ab)
https://staging.problemroulette.ai.umich.edu/AES220/SessionHistory/156388/

Title: WCAG 1.3.1: <ul> and <ol> must only directly contain <li>, <script> or <template> elements (.card-list)
Tags: Accessibility, WCAG 1.3.1, list

Issue: <ul> and <ol> must only directly contain <li>, <script> or <template> elements (list - https://accessibilityinsights.io/info-examples/web/list)

Target application: Problem Roulette - https://staging.problemroulette.ai.umich.edu/console/ReviewStudentFeedback

Element path: .card-list

Snippet: <ul data-v-206a9887="" class="card-list feedbacks">

Fix the following:
List element has direct children that are not allowed: span

====

This accessibility issue was found using Accessibility Insights for Web 2.38.1 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
![image](https://github.com/zcjess/Problem-Roulette/assets/90011962/68122bfa-0e2f-479d-a77f-c7837e66020b)

**Comment by zcjess** [comment]
Title: WCAG 1.3.1: <li> elements must be contained in a <ul> or <ol> (li)
Tags: Accessibility, WCAG 1.3.1, listitem

Issue: <li> elements must be contained in a <ul> or <ol> (listitem - https://accessibilityinsights.io/info-examples/web/listitem)

Target application: Problem Roulette - https://staging.problemroulette.ai.umich.edu/AES220/SessionHistory/156388/

Element path: div > li

Snippet: <li>

Environment: Chrome version 114.0.0.0

====

Fix the following:
List item does not have a <ul>, <ol> parent element

![image](https://github.com/zcjess/Problem-Roulette/assets/90011962/050564e7-ee5c-463d-9df8-2a04c21c89bb)
Title: WCAG 1.3.1: <li> elements must be contained in a <ul> or <ol> (li[data-v-206a9887=""]:nth-child(1))
Tags: Accessibility, WCAG 1.3.1, listitem

Issue: <li> elements must be contained in a <ul> or <ol> (listitem - https://accessibilityinsights.io/info-examples/web/listitem)

Target application: Problem Roulette - https://staging.problemroulette.ai.umich.edu/console/ReviewStudentFeedback

Element path: li[data-v-206a9887=""]:nth-child(1)

Snippet: <li data-v-de27697c="" data-v-206a9887="">

Fix the following:
List item does not have a <ul>, <ol> parent element

====

This accessibility issue was found using Accessibility Insights for Web 2.38.1 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
![image](https://github.com/zcjess/Problem-Roulette/assets/90011962/d3c72505-d6bc-43d9-a4a4-0cce6df53941)

**Comment by zcjess** [comment]
Title: WCAG 1.3.1: <li> elements must be contained in a <ul> or <ol> (li:nth-child(1))
Tags: Accessibility, WCAG 1.3.1, listitem

Issue: <li> elements must be contained in a <ul> or <ol> (listitem - https://accessibilityinsights.io/info-examples/web/listitem)

Target application: Problem Roulette - https://staging.problemroulette.ai.umich.edu/console/AES220/CourseAnalytics/

Element path: .dropdown-menu > li:nth-child(1)

Snippet: <li> <a class="active course-navitem" href="/console/AES220/CourseAnalytics/"> <img class="course-icon icon" src="/static/img/icons/stats-250.png"> AES220 </a> </li>

Fix the following:
List item does not have a <ul>, <ol> parent element without a role, or a role="list"

====

This accessibility issue was found using Accessibility Insights for Web 2.38.1 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
![image](https://github.com/zcjess/Problem-Roulette/assets/90011962/0b241df6-d3d5-48c5-81ac-e34d02588c12)

**Comment by zcjess** [comment]
List item does not have a <ul>, <ol> parent element without a role, or a role="list"
![image](https://github.com/zcjess/Problem-Roulette/assets/90011962/57f6f45d-acd4-4a6d-813b-4d8e9ad59a40)
https://staging.problemroulette.ai.umich.edu/AES220/SessionCreation/?session_type=group-study#/GroupStudy#top
